### PR TITLE
Updated ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by shikhar-scs**
As suggested by @gideonthomas at #905, the issue template has now been updated to the new format.

Please review @gideonthomas

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,18 +1,2 @@
...
-**Expected behavior:** [What you expected to happen]
-
-**Actual behavior:** [What actually happened]
-
-### Versions
-
-Please include the browser name and version you are using.
-Please include the OS (macOS, Windows, Linux) you are using and the version of the OS you're running.
+We track our issues in the Thimble repository. 
+Please file an issue by going to this link -  https://github.com/mozilla/thimble.mozilla.org/issues/new
```
Comment (gideonthomas) [inline ISSUE_TEMPLATE.md]: sorry @shikhar-scs, could you also add ` and mention that the issue is with the Brackets module` to the end of this sentence?

Comment (shikhar-scs) [reply ISSUE_TEMPLATE.md]: @gideonthomas Yeah Sure

**Comment by shikhar-scs** [comment]
@gideonthomas I've incorporated the changes, should I also squash or is it ok?

**Comment by shikhar-scs** [comment]
Thanx @gideonthomas

**Pull request merged**